components/list/list.js

```diff
@@ -74,6 +74,20 @@ class List extends SelectionMixin(LitElement) {
 		return this._getItems().indexOf(item);
 	}
 
+	getSelectionInfo(includeNested) {
```

Comment:
This override is necessary because determining the keys is special for lists.  I considered ways of relating the nested selection-inputs to the parents (note they are not actually parents, or even in the same DOM tree) using IDs but it gets pretty messy and fragile. So, I eventually arrived at this sol'n which is pretty simple.